interfaces/effect_player.hpp

```diff
@@ -10,6 +10,7 @@ namespace bnb::oep::interfaces
 }
 
 using effect_player_sptr = std::shared_ptr<bnb::oep::interfaces::effect_player>;
+using oep_eval_js_result_cb_sptr = std::shared_ptr<std::function<void(const std::string&)>>;
```

Comment:
Better to use:
using oep_eval_js_result_cb = std::function<void(const std::string&)>;